cmd/vicadmin/logs.go

```diff
@@ -378,14 +378,15 @@ func findSeekPos(f *os.File) int64 {
 		// Go back nBytes from the last newline we've seen (or beginning-of-file),
 		// and read the next nBytes
 		readPos := lastnl - int64(len(buf))
```

Comment:
why are we reading from the location of the last newline? We should be using the last value of readPos.

Work through the current logic when you've got a buffer sized 1000, a file sized 2000 and no newline in it:
1. readPos = 2000 - 1000 = 1000
2. bufend = 1000
3. off = bufend = 1000
3a. in loop: off = -1 (not found)
4. lastnl = readPos + bufend = 1000 + 1000 = 2000

And that's an infinite loop.